src/hook_files/cli.sh

```diff
@@ -26,7 +26,7 @@ installRustyHookCli() {
 }
 
 ensureMinimumRustyHookCliVersion() {
-  currentVersion=$(rusty-hook -v)
+  currentVersion=$(rusty-hook -V | awk '{ print $2 }')
```

Comment:
I can't think of a system I've been on that didn't have `awk` but do you know of any that don't include it by default? Does it come with git for windows?

Reply:
I think so: https://github.com/git-for-windows/git-sdk-64/search?q=awk

Reply:
Pretty sure it's usually safe to assume any environment running a `.sh` script has standard GNU utilities

Reply:
> Pretty sure it's usually safe to assume any environment running a `.sh` script has standard GNU utilities

One of the things that was really eye opening for me back during some of the RLS/RustAnalyzer discsussions a while back was the vast array of scenarios folks had for their inner dev loops (especially for embedded/IoT dev), many of which I'd never really thought about before.

I agree this should be safe and am content with it, but do think it 's worthwhile to check these sorts of things preemptively. End users don't really care all that much whether lib Foo or lib Bar is used internally for cli functionality by a tool they are using, but they do care if a change breaks their workflows :smile:

Reply:
I guess i'll just add `-v` back

Reply:
That's not necessary (unless you're thinking we can customize the version output for all version-related flags). This will be fine, just wanted to make sure there weren't any noteworthy exceptions where this could be problematic

Reply:
Yeah, but `-V` prints out something like "rusty-hook 0.1.2"

I added `-v` which just prints out CARGO_PKG_VERSION and nothing more.

Reply:
SGTM, thanks!